pkg/k8s/informers.go

```diff
@@ -78,8 +79,11 @@ const (
 const (
 	// DefaultKubeEventResyncInterval is the default resync interval for k8s events
 	// This is set to 0 because we do not need resyncs from k8s client, and have our
-	// own Ticker to turn on periodic resyncs.
+	// own Ticker to turn on periodic resyncs for the MeshConfig resource.
 	DefaultKubeEventResyncInterval = 0 * time.Second
+	// MRCResyncInterval is the resync interval to provide a retry mechanism for MRC event handlers
+	// It is half of the MrcDurationPerStage (the amount of time we leave each MRC in a stage before moving to the next stage)
+	MRCResyncInterval = certificate.MrcDurationPerStage / 2
```

Comment:
how was 2.5 mins chosen?

Reply:
Talked with @steeling and decided 1/2 of what we recommend for setting the transition time.